Backpack/Rating/Classes/BPKRating.swift

```diff
@@ -0,0 +1,252 @@
+        }
+    }
+    public var subtitle: String? {
+        get {
+            subtitleLabel.text ?? ""
+        }
+        set {
+            subtitleLabel.text = newValue
+            updateLookAndFeel()
+        }
+    }
+
+    // MARK: - Subviews
+    private let horizontalStackView: UIStackView = {
+        let horizontalStack = UIStackView()
+        horizontalStack.axis = .horizontal
+        horizontalStack.spacing = BPKSpacingMd
+        horizontalStack.alignment = .lastBaseline
+        return horizontalStack
+    }()
+
+    private let ratingValueAndScaleStackView: UIStackView = {
+        let horizontalStack = UIStackView()
+        horizontalStack.axis = .horizontal
+        horizontalStack.alignment = .lastBaseline
+        return horizontalStack
+    }()
+
+    private let titleSubtitleStackView: UIStackView = {
+        let verticalStack = UIStackView()
+        verticalStack.axis = .vertical
+        verticalStack.alignment = .leading
+        verticalStack.distribution = .fill
+        return verticalStack
+    }()
+
+    private let ratingValueLabel = BPKLabel(fontStyle: .textLabel1)
+
+    private let ratingScaleLabel: BPKLabel = {
+        let label = BPKLabel(fontStyle: .textCaption)
+        label.textColor = BPKColor.textSecondaryColor
+        return label
+    }()
+
+    private let titleLabel: BPKLabel = {
+        let label = BPKLabel(fontStyle: .textLabel1)
+        label.textColor = BPKColor.textPrimaryColor
+        return label
+    }()
+
+    private let subtitleLabel: BPKLabel = {
+        let label = BPKLabel(fontStyle: .textBodyDefault)
+        label.textColor = BPKColor.textSecondaryColor
+        return label
+    }()
+
+    private var titleViewBottomConstraint: NSLayoutConstraint?
+
+    // MARK: - Init
+    public init(
+        accessibilityLabel: String,
+        title: String,
+        value: Float,
+        ratingScale: BPKRatingScale = .zeroToFive,
+        size: BPKRatingSize = .default,
+        subtitle: String? = nil,
+        showScale: Bool = true
+    ) {
+        self.value = value
+        self.ratingScale = ratingScale
+        self.size = size
+        self.showScale = showScale
+        self.titleView = titleLabel
+        super.init(frame: .zero)
+
+        self.title = title
+        self.subtitle = subtitle
+        self.accessibilityLabel = accessibilityLabel
+
+        setup()
+        updateLookAndFeel()
+    }
+
+    public init(
+        accessibilityLabel: String,
+        value: Float,
+        ratingScale: BPKRatingScale = .zeroToFive,
+        size: BPKRatingSize = .default,
+        subtitle: String? = nil,
+        showScale: Bool = true,
+        titleView: UIView
+    ) {
+        self.value = value
+        self.ratingScale = ratingScale
+        self.size = size
+        self.showScale = showScale
+        self.titleView = titleView
+
+        super.init(frame: .zero)
+
+        self.subtitle = subtitle
+        self.accessibilityLabel = accessibilityLabel
```

Comment:
We need to set this as an 'accessibilityElement' so the children are ignored. 
```suggestion
        self.isAccessibilityElement = true
        self.accessibilityLabel = accessibilityLabel
```

Reply:
update: 95b3ab64bb6d9276184cc0fdac34f6005349f776

This is the update voice over behaviour.

Reply:
https://user-images.githubusercontent.com/102601094/220103237-06e5353d-5bf1-4217-aaa2-73de9590bc84.mov